app/forms/journeys/further_education_payments/building_construction_courses_form.rb

```diff
@@ -0,0 +1,65 @@
+module Journeys
+  module FurtherEducationPayments
+    class BuildingConstructionCoursesForm < Form
+      include ActiveModel::Validations::Callbacks
+      include ActionView::Helpers::UrlHelper
+      include ActionView::Helpers::OutputSafetyHelper
+      include GovukVisuallyHiddenHelper
+      include GovukLinkHelper
+
+      attribute :building_construction_courses, default: []
+
+      before_validation :clean_courses
+
+      validates :building_construction_courses,
+        presence: {message: i18n_error_message(:inclusion)},
+        inclusion: {in: ->(form) { form.checkbox_options.map(&:id) }, message: i18n_error_message(:inclusion)}
+
+      def course_field
+        :building_construction_courses
+      end
+
+      def checkbox_options
+        [
+          OpenStruct.new(
+            id: "esfa_buildingconstruction",
+            name: "ESFA-funded qualifications at level 3 and below in the #{govuk_link_to "building and construction", "https://www.qualifications.education.gov.uk/Search?Status=Approved&Level=0,1,2,3,4&Sub=7"} sector subject area"
```

Comment:
i tried a few different approaches to get link into label and personally found this one to be the best. if you don't fancy this approach please suggest a better approach

Reply:
ok, i've now moved all of these in to i18n